src/Castle.Services.Logging.SerilogIntegration/Castle.Services.Logging.SerilogIntegration.csproj

```diff
@@ -25,7 +25,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Serilog" Version="2.0.0" />
+		<PackageReference Include="Serilog" Version="3.0.0" />
```

Comment:
I'm not sure whether this dependency update should be mixed up with the changes to the targeted platforms.

Leave it as is for now... I may or may not get back to you about this later.

Reply:
restored